src/de/jost_net/JVerein/gui/control/MitgliedskontoControl.java

```diff
@@ -551,6 +552,13 @@ public TablePart getMitgliedskontoList(Action action, ContextMenu menu, boolean
       mitgliedskontoList.addColumn("Zweck", "zweck1");
       mitgliedskontoList.addColumn("Betrag", "betrag",
           new CurrencyFormatter("", Einstellungen.DECIMALFORMAT));
+      mitgliedskontoList.addColumn("Zahlungsweg","zahlungsweg", new Formatter() {
+        @Override
+        public String format(Object o)
+        {
+          return new Zahlungsweg((Integer)o).getText();
+        }
+      });
```

Comment:
Könnte man hier noch die Buchungsklasse und die Buchungsart anzeigen?
Bei der Buchungsklasse abhängig vom Schalter in den Einstellungen, siehe Tabelle bei den Buchungen.

Reply:
habe ich gemacht